daemon/runtime_unix.go

```diff
@@ -134,3 +137,35 @@ func (daemon *Daemon) getRuntime(name string) (*types.Runtime, error) {
 
 	return rt, nil
 }
+
+// isValidC8dRuntimeName tests whether name
+// is a well-formed ContainerD runtime name,
```

Comment:
note that the canonical name for containerd is all lowercase (`containerd`)

Reply:
🤦 I must've mixed it up with runC

Reply:
I don't think there's an "official" statement, but afaik, it's all lowercase (but I know _some_ people use the capital D) - as we're using lowercase everywhere, best to stick with it 😅